README.md

```diff
@@ -200,4 +201,14 @@ NULL ...
 ### Internals
 ![](./assets/internals.png)
 
+## Release versioning
+The release version of django-spanner in [setup.py](./setup.py) SHOULD follow this form
```

Comment:
```suggestion
The release version of django-spanner in [setup.py](./setup.py) follows this format:
```

Reply:
I'd encourage we include the `SHOULD` as it is the way that django-backends must be versioned.

Reply:
Ok in that case, "must" but there's no need to capitalize it.